module-2/backend/week_one.md

```diff
@@ -5,30 +5,83 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  GET - show a record from the database
+  POST - create a record in the database
+  PUT - update a record in the database
+  DELETE - remove a record from database
+
 2. What is Sinatra?
+  a framework that starts a server, and makes it simple to manage routes
+
 4. What is MVC?
+  convention from rails that is a standard template for organizing web app paths
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  in some cases it's necessary so that files can find each other, otherwise it's to get ready for rails

```

Comment:
I would also say that it makes it so that we can get into a new project that someone else has been developing and quickly get up to speed/start contributing. The same going the other way: people can come help us with our application and pretty quickly make meaningful contributions.